.travis.yml

```diff
@@ -19,9 +18,10 @@ matrix:
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.*
-  allow_failures:
-    - php: hhvm
+      env: DEPENDENCIES="symfony/lts:^2"
+    - php: 7.1
+      env: DEPENDENCIES="symfony/lts:^3"
+    - php: 7.1
```

Comment:
why duplicating the 7.1 job ?

Reply:
Actually, this one tests Symfony 4, while the previous is for 3.x

Reply:
this one is *exactly* the same than the job part of the normal matrix above.

Reply:
And actually, I'm wrong, both tests Symfony 4 :/